book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_reader :rental
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
```

Comment:
- [ ] Please initialize a `@rental = [ ]` variable on the constructor where you save your rentals on line 12.

Reply:
Of course, 
Thank you @thecodechaser